### PR TITLE
Use st2 pack install style

### LINKED ISSUE #366: Add tests for new pack management commands
```sh
st2 pack install
st2 pack list
...
```
and so on in https://github.com/StackStorm/st2-packages/blob/master/rake/spec/default/70-st2_actions-integrity_spec.rb

### PR DESCRIPTION
**Pull request by warrenvw**
Fixes #366

### CODE REVIEW

File: rake/spec/default/70-st2_actions-integrity_spec.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
     its(:exit_status) { is_expected.to eq 0 }
   end
 
-  describe command("st2 run packs.install packs=github") do
+  describe command("st2 packs install github") do
```
Comment (Mierdin) [inline rake/spec/default/70-st2_actions-integrity_spec.rb]: This should be `st2 pack install github`

Comment (Mierdin) [inline rake/spec/default/70-st2_actions-integrity_spec.rb]: This should be `st2 pack install github`

Comment (warrenvw) [inline rake/spec/default/70-st2_actions-integrity_spec.rb]: ack! thanks.

**Comment by armab** [comment]
Hm, I'm curious why CI is not running and there is no status.

**Comment by armab** [comment]
Forked PR build was disabled: https://i.imgur.com/0D86UCx.png
Re-enabled it.

**Pull request merged**